recipes/_install.rb

```diff
@@ -31,14 +33,14 @@
   source node['looker']['startup_script_url']
   owner 'looker'
   group 'looker'
-  mode 0750
+  mode 0o750
```

Comment:
@davidsayers What does this format do vs just `0750` out of interest?

Reply:
ummm, rubocop did it :-)

Think it's an octal thing.